openApi/openApi.yaml

```diff
@@ -0,0 +1,46 @@
+openapi: 3.0.3
```

Comment:
I added a `docs` folder on my branch with the asyncapi spec as well. Can maybe move this there once rebased? :)

Reply:
moved to the docs folder